src/main/java/org/broadinstitute/hellbender/tools/PrintReads.java

```diff
@@ -2,21 +2,29 @@
 
 import org.broadinstitute.barclay.argparser.Argument;
 import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.help.DocumentedFeature;
 import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
 import org.broadinstitute.hellbender.cmdline.programgroups.ReadProgramGroup;
 import org.broadinstitute.hellbender.engine.FeatureContext;
 import org.broadinstitute.hellbender.engine.ReadWalker;
 import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.utils.help.HelpConstants;
 import org.broadinstitute.hellbender.utils.read.GATKRead;
 import org.broadinstitute.hellbender.utils.read.SAMFileGATKReadWriter;
 
 import java.io.File;
 
+/**
+ * Print reads from a SAM/BAM/CRAM file.
+ *
+ * {@GATK.walkertype ReadWalker}
```

Comment:
Its part of the doc system. The doc system queries the doclet for a custom prefix ("GATK" in this case) and then pulls any embedded custom javadoc tags with that prefix up into the Freemarker map. It has some advantages and some disadvantages, but it allows the doc to be changed without modifying or depending on code. For now, its how we're adding the "walkertype" category to the doc.